runnable/test34.d

```diff
@@ -707,6 +710,7 @@ void foo35()
         c = 3;
 
         xxx = cast(typeof(xxx))(a + b);
+	version (DMD_InlineAsm)
         asm { int 3; }
```

Comment:
That makes sense, I did not think of it.

I do not know what the foo35() is testing here.